### PR TITLE
0.1 Release Alpha Bug Fixes

### PR DESCRIPTION
**Pull request by VespertinePetrichor**
## Changelog:
- Updated version history
- Commented invalid effects, modifiers, vars, achievements and events.
- Removed BAK and duplicate files.
- Fixed broken triggers, effects, events
  - knight_prestige_gain_on_kill_inverse
  - war_reparation_defeat
  - has_saint_relics_trigger
  - fund_inspiration.5021
  - court.0108
- Adapted religious conversion costs for wc religions
- Updated geographical regions list
- History:
  - Alterac and Dragon birth history errors
  - Corrected null title history
  - Corrected invalid capital provinces
- Check commit history for full comments.

## Tests:
- [x] There are no errors in `wc` files in `Documents\Paradox Interactive\Crusader Kings III\logs\error.log` except `portrait_decals.cpp:101`
- [x] The mod takes less than 5.5 GB in the Task Manager (Windows)

# How to test:
Check for crashing.

### CODE REVIEW

**Review by TheGroxEmpire — COMMENTED** [review]
All three bookmark crashed.
First bookmark crashed at December 604.
Second bookmark crashed at December 740.
Third bookmark crashed at January 737 right when saving game.

[first_bookmark_25-03-2023-0231.zip](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/files/11066433/first_bookmark_25-03-2023-0231.zip)

[second_bookmark_25-03-2023-0300.zip](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/files/11066434/second_bookmark_25-03-2023-0300.zip)

[third_bookmark_25-03-2023-0336.zip](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/files/11066436/third_bookmark_25-03-2023-0336.zip)

Also it's worth to mention that Ner'zhul is still alive by the time the game crashed on 2nd and 3rd bookmark. The Dreadlord is still there also. There's a possibility that the event to destroy him didn't fire.

On 2nd bookmark. The Lich King lost most of his vassal but still has Arthas. Arthas never became the Lich King on that bookmark.

**Comment by ValianBlue** [comment]
_Testing the first bookmark, I crashed on 1 January, 594._

[594 Crash.zip](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/files/11106274/594.Crash.zip)

File: common/script_values/00_basic_values.txt [hunk 1]
```diff
@@ -615,8 +615,14 @@ knight_prestige_gain_on_kill_inverse = {
 	}
 	if = {
 		limit = {
-			root = {
-				is_lowborn = yes
+			#Warcraft
```
Comment (sumankun) [inline common/script_values/00_basic_values.txt:618]: trigger if needs limit

**Review by sumankun — APPROVED** [review]
seems ok

**Pull request merged**